contrib/postgres_fdw/deparse.c

```diff
@@ -3101,13 +3101,10 @@ deparsePartialAggFunctionParamFilter(Aggref *node, deparse_expr_cxt *context,
 	{
 		case 2100:
 			/* int8 AVG */
-			/* Fall through */
```

Comment:
Why is it necessary to remove these comments in this file?

Reply:
You said in the previous patch that there's no need to annotate those cases which don't have any code, so I removed it